### DISCUSSION TITLE
Some questions about applying for GSoC

### DISCUSSION THREAD

**Discussion by sususama** [openvinotoolkit/openvino_notebooks#963] [Q&A] [answered]
Hi, @OpenVINO-dev-contest 
I would like to ask how many people apply for your project now, is there any Pretest? Because there is no contact information such as email on your GitHub homepage, please provide your email if it is convenient for me to contact you.

**Comment by openvino-dev-samples** [comment, marked as answer]
Hi @zhuo-yoyowz do you know the number of application?
Hi @sususama For pretest information: https://github.com/openvinotoolkit/openvino_notebooks/issues/832
You can send your content to both ```ethan.yang@intel.com``` and ```zhuo.wu@intel.com```

**Comment by sususama** [comment]
Hi, @OpenVINO-dev-contest @zhuo-yoyowz 
I've done the community's pretest: #950 
Are there any other requirements besides this?

**Reply by openvino-dev-samples** [reply]
Besides it, I believe there is no other pre-requirement for this task

**Reply by sususama** [reply]
Hi, @OpenVINO-dev-contest @zhuo-yoyowz 

I have emailed both teachers, wondering if you both received it? If you have received it, please reply me, thank you very much.
